### ISSUE TITLE
Appstream error or warning?

### ISSUE THREAD

**Issue by yakushabb** [flathub-infra/flatpak-builder-lint#817]
I use the following command to check the flatpak-builder-lint results.

```
  LC_ALL=C flatpak run --command=flatpak-builder-lint org.flatpak.Builder repo repo
```

It creates a json result something like the following:

```json
{
    "errors": [
        "appstream-external-screenshot-url"
    ],
    "warnings": [
        "appstream-screenshot-missing-caption"
    ],
    "appstream": [
        "W:APP.metainfo.xml:screenshot-default-missing:18 No screenshot is marked as default."
    ],
    "info": [
        "appstream-failed-validation: Metainfo file APP.metainfo.xml has failed validation. Please see the errors in appstream block"
    ],
    "message": "See https://docs.flathub.org/linter for details and exceptions"
}

```

There are errors and warnings blocks. They are clear.

But the appstream part is a bit confusing. It does not say it's an error. It starts with a W, presume its a warning. 

But vorarbeiter says.

```bash
Error: Appstream: 'W:APP.metainfo.xml:screenshot-default-missing:32 No screenshot is marked as default.
```

Could you make it a bit clearer. 

Thanks.

**Comment by bbhtt** [comment]
Appstream considers both warnings and errors as errors ie. it fails.
